docs/deployment/engine_on_yarn.md

```diff
@@ -175,6 +175,24 @@ If the `TopSpeedWindowing` passes, configure it in `$KYUUBI_HOME/conf/kyuubi-env
 $ echo "export HADOOP_CONF_DIR=/path/to/hadoop/conf" >> $KYUUBI_HOME/conf/kyuubi-env.sh
 ```
 
+#### Required Environment Variable
+
+The `FLINK_HADOOP_CLASSPATH` is required, too. It should contain `commons-collections-*.jar`, 
+`hadoop-client-runtime-*.jar`, `hadoop-client-api-*.jar` and `htrace-core4-*.jar`.
+All four jars are in the `HADOOP_HOME`. 
+
+For example, in Hadoop 3.1.0 version, the following is their location. 
```

Comment:
Can we demo with the version of Hadoop which Kyuudi bundled?

Reply:
OK I will change it